### PR TITLE
Add smart contracts deployment addresses for `testnet` and `mainnet`

### PR DESCRIPTION
**Pull request by matjazv**
- Smart contracts addresses for `testnet` and `mainnet` deployments were added as `.json` files.
- 1st deployment script only deletes files from a selected network folder and not deletes whole `deployment` folder.

### CODE REVIEW

File: deployment/mainnet/l2addresses.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "L2ClaimContract": "0x0000000000000000000000000000000000000000",
```
Comment (sergeyshemyakov) [inline deployment/mainnet/l2addresses.json:2]: Any reason not to put [0xD7BE2Fd98BfD64c1dfCf6c013fC593eF09219994](https://blockscout.lisk.com/address/0xD7BE2Fd98BfD64c1dfCf6c013fC593eF09219994) here?

File: deployment/mainnet/l2addresses.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "L2ClaimImplementation": "0x0000000000000000000000000000000000000000",
```
Comment (sergeyshemyakov) [inline deployment/mainnet/l2addresses.json:3]: Any reason not to put [0x60790Dc2d45BaA8B36282889569BbB301F4D0c41](https://blockscout.lisk.com/address/0x60790Dc2d45BaA8B36282889569BbB301F4D0c41) here?

**Review by sergeyshemyakov — CHANGES_REQUESTED** [review]

File: deployment/mainnet/l2addresses.json (hunk 1 above)
Comment (matjazv) [reply deployment/mainnet/l2addresses.json]: Thanks for catching this!
Looks like I've discarded changes once L2Claim contract was deployed.
Those files are automatically generated so in normal case they don't need to be modified by hand.

File: deployment/mainnet/l2addresses.json (hunk 2 above)
Comment (matjazv) [reply deployment/mainnet/l2addresses.json]: Same as above.
Done.

**Review by Phanco — APPROVED** [review]
👍

**Pull request merged**